cli/command/format.go

```diff
@@ -132,10 +134,35 @@ func displayFormatStatus(curveadm *cli.CurveAdm, fcs []*configure.FormatConfig,
 }
 
 func runFormat(curveadm *cli.CurveAdm, options formatOptions) error {
-	// 1) parse format config
-	fcs, err := configure.ParseFormat(options.filename)
-	if err != nil {
-		return err
+	var err error
+	var fcs []*configure.FormatConfig
+	diskRecords := curveadm.DiskRecords()
+
+	// 1) parse format config from yaml file or database
+	if len(diskRecords) == 0 {
+		fcs, err = configure.ParseFormat(options.filename)
+		if err != nil {
+			return err
+		}
+	} else {
+		for _, dr := range diskRecords {
+			containerImage := configure.DEFAULT_CONTAINER_IMAGE
+			if len(dr.ContainerImageLocation) > 0 {
```

Comment:
Why use `ContainerImageLocation` instead of `ContainerImage`?

Reply:
We're considering to use local container image instead of deploying docker registry in a local network environment.